test/e2e/inspect_test.go

```diff
@@ -55,7 +55,7 @@ var _ = t.Describe("inspect", func() {
 
 	AfterEach(func() {
 		Expect(t.Crictl("rmp -f " + sandbox)).To(Exit(0))
-		Expect(t.Crictl("rmi " + imageLatest)).To(Exit(0))
+		Expect(t.Crictl("rmi -a")).To(Exit(0))
```

Comment:
will it affect any other tests this way? Anything that needs images and running in-parallel would be affected?

Reply:
Yes, I'll adapt the tests to only remove the images being used as well as the pause image. I change the other tests according to this as well.